sycl/test/basic_tests/fp-accuracy.cpp

```diff
@@ -0,0 +1,15 @@
+// RUN: %clangxx -%fsycl-host-only -c -ffp-accuracy=high -faltmathlib=SVMLAltMathLibrary -fno-math-errno %s
```

Comment:
```suggestion
// RUN: %clangxx -fsycl-host-only -c -ffp-accuracy=high -faltmathlib=SVMLAltMathLibrary -fno-math-errno %s
```

Reply:
The % is needed. The RUN line will be expanded to -Xclang -fsycl-is-host

Reply:
Thank you, I was wondering why.